index.js

```diff
@@ -105,23 +228,48 @@ function pollMsg () {
             }
           }
         } catch (e) {
-          console.error(JSON.stringify(e))
+          console.error(e)
         }
       }
-      pollMsg()
+      pollMsgIntervel()
     })
   }
 }
+
+/**
+ *  problem/
+ *    If given fb-messenger-gateway is down, this will make a continus request.
+ * In this case this process will consume more power and memory.
+ *  
+ *  way/
+ *    If facebook messenger gateway is down, his will make a request certain intervel.
+ * intervals are 5, 10, 15, 20, 25 and 30 seconds else we will poll a message immediately. 
+ *
+ */
+function pollMsgIntervel(){
+  if(pollReq.error && pollReq.error.errno){
```

Comment:
Why should there be an errno? Does it represent a certain type of error? This is not clear

Reply:
sure, I will make changes